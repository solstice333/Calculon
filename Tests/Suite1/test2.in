This is test 2!
This is another line.
