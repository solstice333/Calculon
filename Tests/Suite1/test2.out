Hello, world!
0: ./a.out
1: -xyz
Writing input contents:
This is test 2!
This is another line.
